draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1114,7 +1114,31 @@ The work on this draft was started at OAuth Security Workshop 2022 in Trondheim,
 
 # IANA Considerations {#iana_considerations}
 
-TBD
+## JSON Web Token Claims Registration
+
+This specification requests registration of the following Claims in the
+IANA "JSON Web Token Claims" registry [@IANA.JWT] established by [@!RFC7519].
+
+Object Property Disclosure Digests:
+
+*  Claim Name: `_sd`
+*  Claim Description: Object property disclosure digests
```

Comment:
```suggestion
*  Claim Description: Disclosure digests
```
I am sorry, i could not understand `Object property disclosure digests`..? did you want to say that digests are hashes over objects?

Reply:
sure, if that's more understandable